crates/viewer/re_ui/src/ui_ext.rs

```diff
@@ -30,6 +30,21 @@ pub trait UiExt {
         response.on_hover_text(error_text)
     }
 
+    /// Shows an error label with the entire error text and copies the text to the clipboard on click.
+    ///
+    /// Use this only if you have a lot of space to spare.
+    /// Don't start the string with "Error: " since this is added automatically.
+    fn error_label_long(&mut self, error_text: &str) -> egui::Response {
+        let label = egui::Label::new(self.ui().ctx().error_text(format!("Error: {error_text}")))
+            .selectable(false)
+            .sense(egui::Sense::click());
+        let response = self.ui_mut().add(label);
+        if response.clicked() {
+            self.ui().ctx().copy_text(error_text.to_owned());
```

Comment:
This is a nice touch, but there is no affordance that tells the user that they can copy the error with a click.

@gavrelina may have a better idea here, but some alternatives:

* Show "Click to copy" in a tooltip on hover
* Make the text selectable instead
* Add a 📋 button next to the message instead

Reply:
for context: I copied this from the "small" error text method above. But I do agree!

Reply:
our `📋` icon is too ugly for this (also tried adding it to the regular `error_label`, but it's just too ugly). going with the minimum here for now whcih is selectable + hover